problems/1431.py

```diff
@@ -0,0 +1,25 @@
+class Solution(object):
+    def kidsWithCandies(self, candies, extraCandies):
+        """
+        :type candies: List[int]
+        :type extraCandies: int
+        :rtype: List[bool]
+        """
+        #find the max number of candy
+        # for each element add extraCandies
+        maxNumberOfCandy = 0
```

Comment:
What about negatives? Have u considered using math.min instead? This is the smallest number integer supports. Negative candies doesn’t really make sense but I think should know this for future interviews

Reply:
I think u mean minint which python doesn't have. Python has this instead `-sys.maxint - 1` which I will use for future code